test/Controllers/UsersTest.php

```diff
@@ -69,80 +69,101 @@ public function testCreateUser() {
+
+        // XXX Can't delete created users ATM.
     }
 
     /**
      * Test GET /users/{id}
      * GonebusyLib\Controllers\UsersController::getUserById()
      */
     public function testGetUserById() {
-        // $id = 'id';
-        // $response = $this->users->getUserById(
-        //     $this->authorization,
-        //     $id);
-        // // print_r($response);
-        //
-        // $this->assertObjectHasAttribute('user', $response);
-        // $this->assertEquals($id, $response->user->accountManagerId);
+        // Create user:
+        $createUserBody = $this->uniqueUserData();
+        $createResponse = $this->users->createUser($this->authorization, $createUserBody);
+
+        // Get user by it's id
+        $response = $this->users->getUserById(
+            $this->authorization,
+            $createResponse->user->id);
+        // /* DEBUG */ print_r($response);
+
+        // Was it fetched?
+        $this->assertObjectHasAttribute('user', $response);
+
+        // Does it have all the original data we sent?
+        $responseBody = $this->dataFromResponse($response);
+        $this->assertEquals($responseBody, $createUserBody);
     }
 
     /**
      * Test PUT /users/{id}
      * GonebusyLib\Controllers\UsersController::updateUserById()
      */
-    public function testUpdateUserById() {
-        // // $this->expectException(EntitiesErrorException::class);
-        //
-        // $id = 'id';
-        // $updateUserByIdBody = new UpdateUserByIdBody();
-        // $response = $this->users->updateUserById(
-        //         $this->authorization,
-        //         $id,
-        //         $updateUserByIdBody);
-        // print_r($response);
-
-        // $this->assertObjectHasAttribute('user', $response);
-        // $this->assertEquals($id, $response->user->accountManagerId);
+    public function testUpdateUserById() { // $this->expectException(EntitiesErrorException::class);
+        // Create user:
+        $createUserBody = $this->uniqueUserData();
+        $createResponse = $this->users->createUser($this->authorization, $createUserBody);
+        // /* DEBUG */ print_r($createResponse);
+
+        $anotherUserBody = $this->uniqueUserData();
+
+        // Update user by it's id
+        $response = $this->users->updateUserById(
+            $this->authorization,
+            $createResponse->user->id,
+            $anotherUserBody);
+        // /* DEBUG */ print_r($response);
+
+        // Was it fetched?
+        $this->assertObjectHasAttribute('user', $response);
+
+        // Does it have all the new data we sent?
+        $responseBody = $this->dataFromResponse($response);
+        $this->assertEquals($responseBody, $anotherUserBody);
+
+        // XXX Do we need to getUserById again and/or check the ids?
     }
 
     /**
      * Test GET /users
      * GonebusyLib\Controllers\UsersController::getUsers()
      */
     public function testGetUsers() {
-
+        $perPage = 3; // XXX Are this enough?
```

Comment:
@alexagranov are 3 elements good enough? I think the fewer the better since it's connecting to the actual API.